scalafix-docs/src/main/scala/docs/RulesModifier.scala

```diff
@@ -0,0 +1,16 @@
+package docs
+
+import mdoc.Reporter
+import mdoc.StringModifier
+import scala.meta.inputs.Input
+
+class RulesModifier extends StringModifier {
+  val name = "scalafix-rules"
+  override def process(
+      info: String,
+      code: Input,
+      reporter: Reporter
+  ): String = {
+    scalafix.website.allRulesTable(reporter)
```

Comment:
As discussed offline, I would suggest to make this a variable, since it doesn't depend on the code block content (which is, in fact, empty)

Reply:
This uses the reporter to warn about missing documentation so I prefer to keep it as a modifier